cosmoz-default-tree.html

```diff
@@ -1,10 +1,21 @@
 <link rel="import" href="cosmoz-tree.html">
+
+<!--
+Navigator through object with treelike datastructure and default settings.
+
+@demo demo/index.html
+-->
 <script>
 	'use strict';
 
+	Polymer({ is: 'cosmoz-default-tree' });
```

Comment:
I'm not sure we actually need to make this a Polymer element.
Did you see a reason?